docs/project/changelog.md

```diff
@@ -58,13 +58,24 @@
   raise a `KeyboardInterrupt` by writing to the interrupt buffer.
   [#1148](https://github.com/iodide-project/pyodide/pull/1148) and
   [#1173](https://github.com/iodide-project/pyodide/pull/1173)
+<<<<<<< HEAD
+- A `JsProxy` of a Javascript `Promise` or other awaitable object is now a 
+  Python awaitable.
+  [#880](https://github.com/iodide-project/pyodide/pull/880)
+- Added a Python event loop to support asyncio by scheduling coroutines to run 
+  as jobs on the browser event loop. This event loop is available by default and 
+  automatically enabled by any relevant asyncio API, so for instance 
+  `asyncio.ensure_future` works without any configuration.
```

Comment:
Should we use create_task here and in tests instead? It is recommended in the docs https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task

Also merge conflicts in this file.

Reply:
I have a bad habit of opening a merge tool in a separate console window and then forgetting to save before I commit.

Reply:
On `create_task`, sounds fine.